.travis.yml

```diff
@@ -1,11 +1,19 @@
 language: php
-php:
-  - 5.2
-  - 5.3
-  - 5.4
-  - 5.5
-  - 5.6
-  - 7
+
+matrix:
```

Comment:
@nikic are you sure about this Travis syntax?  I'm trying to separately use these exact changes to get a clean Travis biuld before applying this PR... but this syntax gives me Ruby builds every time.

Reply:
Weird. This is what the build looked like in my fork: https://travis-ci.com/nikic/PHP_Archive/builds/124732148 Seems to work there.

Reply:
Yeah, that one does look good... let's go with that one as the "good PR build" 👍